test/codeu/chat/util/UpTimeTest.java

```diff
@@ -0,0 +1,43 @@
+package codeu.chat.util;
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import codeu.chat.common.ServerInfo; 
+import java.io.IOException;
+
+
+public class UpTimeTest {
+	Boolean errorThrown = false; 
+	//Tests to see if times are stored correctly in ServerInfo
+  @Test
+  public void testUpTime() {
+	  //creates startTime @ a time 5000ms 
+	Time startTime = Time.fromMs(5000);
+    ServerInfo testServer = new ServerInfo(startTime);
+    assertEquals(5000, testServer.getStartTime().inMs());
+  }
+  
+  @Test
+  public void testUpTime2() {
```

Comment:
minor note: I'd probably use more descriptive function names. What exactly does this test?